src/Nordea/Components/Error.elm

```diff
@@ -0,0 +1,222 @@
+module Nordea.Components.Error exposing (..)
+
+import Css exposing (pct, rem, vh)
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes as Attributes exposing (href)
+import Nordea.Components.FlatLink as FlatLink
+import Nordea.Components.Text as Text
+import Nordea.Html exposing (showIf)
+import Nordea.Resources.Colors as Colors
+import Svg.Styled as Svg exposing (Svg)
+import Svg.Styled.Attributes as SvgAttrs exposing (d)
+
+
+type alias Locale =
+    { no : String, se : String, dk : String } -> String
+
+
+type alias ErrorConfig =
+    { errorType : ErrorType, locale : Locale }
+
+
+type ErrorType
+    = InternalServerError
+    | PageNotFound
+
+
+type Error msg
+    = Error ErrorConfig
+
+
+internalServerError : Locale -> Error msg
+internalServerError locale =
+    init { errorType = InternalServerError, locale = locale }
+
+
+pageNotFound : Locale -> Error msg
+pageNotFound locale =
+    init { errorType = PageNotFound, locale = locale }
+
+
+init : ErrorConfig -> Error msg
+init config =
+    Error { locale = config.locale, errorType = config.errorType }
+
+
+view : List (Attribute msg) -> List (Html msg) -> Error msg -> Html msg
+view attributes children (Error config) =
+    let
+        toI18NString textToTranslate =
+            config.locale textToTranslate
+
+        errorDescription =
+            if config.errorType == PageNotFound then
+                texts.pageNotFound.description |> toI18NString
+
+            else
+                texts.internalServerError.description |> toI18NString
+
+        errorAction =
+            if config.errorType == InternalServerError then
+                texts.internalServerError.action |> toI18NString
+
+            else
+                ""
+    in
+    Html.div
+        [ Attributes.css
+            [ Css.backgroundColor Colors.white
+            , Css.width (pct 100)
+            , Css.height (vh 100)
+            , Css.padding (rem 1.5)
+            , Css.displayFlex
+            , Css.flexDirection Css.column
+            , Css.alignItems Css.center
+            ]
+        ]
+        [ Html.div [ Attributes.css [ Css.marginBottom (rem 1) ] ]
+            [ Text.headlineTwo
+                |> Text.withHtmlTag Html.h1
+                |> Text.view
+                    [ Attributes.css [ Css.textAlign Css.center ]
+                    ]
+                    [ Html.text (texts.heading |> toI18NString) ]
+            ]
+        , Html.div
+            [ Attributes.css
+                [ Css.marginBottom (rem 2)
+                , Css.maxWidth (rem 30)
+                , Css.textAlign Css.center
+                ]
+            ]
+            [ Text.bodyTextLight
+                |> Text.view []
+                    [ Html.text errorDescription
+                    ]
+            ]
```

Comment:
```suggestion
        , Text.bodyTextLight
                |> Text.view 
                     [ Attributes.css
                         [ Css.marginBottom (rem 2)
                         , Css.maxWidth (rem 30)
                         , Css.textAlign Css.center -- Not sure if this is needed since the wrapping div as alignItems center
                         ]
                    ]
                    [ Html.text errorDescription ]
```

Reply:
The wrapping div just align the body text, but the text is not centered if it e.g its shown on two or more lines. So it need textAlign as well